static/src/javascripts/projects/common/modules/atoms/youtube.spec.js

```diff
@@ -48,4 +74,250 @@ describe('youtube', () => {
             expect(document.body.innerHTML).toBe(div);
         }
     });
+
+    describe('muting iframes to trgiger autoplay', () => {
+        it('mutes an iframe by amending the src property if no other URL parameters', () => {
+            const docSrc = 'http://www.testme.com/q';
+
+            const div = `<div id="outerDiv"><iframe id="iframeId" src="${docSrc}"></iframe></div>`;
+
+            if (document.body) {
+                document.body.innerHTML = div;
+            }
+
+            const iframe = ((document.getElementById(
+                'iframeId'
+            ): any): HTMLIFrameElement);
+
+            _.muteIFrame(iframe);
+
+            if (document.body) {
+                expect(iframe.src).toBe('http://www.testme.com/q?mute=1');
+            }
+        });
+
+        it('adds a new parameter to mute iframe if parameters already exist', () => {
+            const docSrc = 'http://www.testme.com/q?randomParam=abc';
+
+            const div = `<div id="outerDiv"><iframe id="iframeId" src="${docSrc}"></iframe></div>`;
+
+            if (document.body) {
+                document.body.innerHTML = div;
+            }
+
+            const iframe = ((document.getElementById(
+                'iframeId'
+            ): any): HTMLIFrameElement);
+
+            _.muteIFrame(iframe);
```

Comment:
(minor / style) We can avoid the underscore here by destructuring the imported `_` somewhere near the top of the file:

```js
const { muteIFrame, getIFrameBehaviour, getIFrameBehaviourConfig } = _;
```

example here:
https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/comment-adverts.spec.js#L44

Reply:
Changed - I was trying to figure out if there was something like this I could do. Should have searched in the codebase!